abixen-platform-web-client/src/main/web/common/navigation/javascript/directives.js

```diff
@@ -23,13 +23,15 @@ platformNavigationDirectives.directive('platformNavigation', ['$log', '$state',
             scope.redirectAction = applicationNavigationItems.getRedirectAction();
             scope.dropdownStyleClass = applicationNavigationItems.getDropdownStyleClass();
             scope.toggle = true;
+            scope.baseUserUrl = "/api/application/users/";
+            scope.avatarUrl = scope.baseUserUrl + "1/avatar/1";
             var mobileView = 992;
 
-            scope.getWidth = function() {
+            scope.getWidth = function () {
```

Comment:
You have formatted everything. So, some lines looks like changed by you, but they are not. Remove formatting of the while file.